src/en/caas.md

```diff
@@ -0,0 +1,192 @@
+kubernetes-worker/0*  active    idle   1        10.40.174.193   80/tcp,443/tcp  Kubernetes worker running.
+  flannel/1           active    idle            10.40.174.193                   Flannel subnet 10.1.25.1/24
+
+Machine  State    DNS            Inst id              Series  AZ  Message
+0        started  10.40.174.204  juju-562a96-0        xenial      Running
+0/lxd/0  started  10.0.45.195    juju-562a96-0-lxd-0  xenial      Container started
+1        started  10.40.174.193  juju-562a96-1        xenial      Running
+
+Relation provider                    Requirer                             Interface         Type         Message
+easyrsa:client                       etcd:certificates                    tls-certificates  regular      
+easyrsa:client                       kubernetes-master:certificates       tls-certificates  regular      
+easyrsa:client                       kubernetes-worker:certificates       tls-certificates  regular      
+etcd:cluster                         etcd:cluster                         etcd              peer         
+etcd:db                              flannel:etcd                         etcd              regular      
+etcd:db                              kubernetes-master:etcd               etcd              regular      
+kubernetes-master:cni                flannel:cni                          kubernetes-cni    subordinate  
+kubernetes-master:kube-api-endpoint  kubernetes-worker:kube-api-endpoint  http              regular      
+kubernetes-master:kube-control       kubernetes-worker:kube-control       kube-control      regular      
+kubernetes-worker:cni                flannel:cni 			  kubernetes-cni    subordinate
+```
+
+### Add the cluster to Juju
+
+We will need some information about the cluster in order to add it to Juju.
+This is found within the main Kubernetes configuration file.
+
+!!! Note:
+    If `conjure-up` was used to install the cluster then the rest of this
+    section can be skipped; this install method adds the cluster for you.
+
+#### Adding quickly (bundle installs)
+
+If the `juju deploy` command was used to deploy the cluster the above file can
+be copied over from the Kubernetes master node (and saved as `~/.kube/config`)
+in this way:
+
+```bash
+mkdir ~/.kube
+juju scp kubernetes-master/0:config ~/.kube/config
+```
+
+We can now take advantage of the `add-k8s` command as it internally parses the
+copied configuration file from the specified path. This allows us to quickly
+add the cluster-cloud, which we have arbitrarily called 'k8cloud':
+
+```bash
+juju add-k8s k8cloud
+```
+
+## Configuration
+
+Juju CAAS applications support application specific configuration. This allows
+k8s configuration to be used to control how Juju deploys the application on
+Kubernetes. The following are supported (these names are the Juju configuration
+attribute names; the k8s meaning should be obvious):
```

Comment:
the k8s meaning is described in the relevant k8s service and ingress resource documentation